scripts/get_baseline_hash.py

```diff
@@ -13,27 +13,34 @@ def parse_arguments():
     )
     return parser.parse_args()
 
+def filter_results(issue):
+    issue_labels = issue['labels']
+    labels_check = True
+    filter_labels = ['build-failure', 'testsuite-failure', 'bisect']
+    for label in issue_labels:
+        if label['name'] in filter_labels:
+            labels_check = False
+            break
+    if 'pull_request' not in issue.keys() and labels_check:
+        return issue
+
 def parse_baseline_hash(url: str, token: str):
     params = {"Accept": "application/vnd.github+json",
               "Authorization": f"token {token}",
               "X-GitHub-Api-Version": "2022-11-28"}
     r = requests.get(url, params=params)
     issues = json.loads(r.text)
-    issue = None
-    for issue in issues:
-        if 'pull_request' not in issue.keys():
-            break
-
+    filtered = list(filter(lambda issue: filter_results(issue), issues))
```

Comment:
nit: Just a preference thing but I find the list-comprehension approach more readable
```
filtered = [issue for issue in issues if filter_results(issue)]
```